### ISSUE TITLE
Ability to preview the website inside the html editor

### ISSUE THREAD

**Issue by sushiwt** [neocities/neocities#477]
To me, having to press the view button every time just to view the site is pretty inconvenient.
![The Neocities HTML Editor buttons (Theme, Save, View, Share)](https://github.com/neocities/neocities/assets/130558752/8bcc5ec6-9f5b-40e0-a6ec-d4190e3f6bfe)

What if there was a way to type in the HTML editor on one side and a way to instantly see the changes you make on the site on the other? Kinda like how CodePen does it.
![A screenshot of codepen.](https://github.com/neocities/neocities/assets/130558752/f06a4e6c-0ad1-4157-bcdf-2b6c4a20c1ff)

That would be appreciated!

P.S. It's kinda already a thing, but only on the Neocities HTML course (albeit, kinda limited), and it'd be awesome if this was carried into the actual editor.
![The Neocities site preview (on the tutorial lol) ](https://github.com/neocities/neocities/assets/130558752/b356aa5c-489b-4bcc-85a0-9655a19eabe8)

**Comment by BanceDev** [comment]
I made a hacky mockup to test this idea:
![image](https://github.com/user-attachments/assets/bd841111-317a-400e-8fc2-4c0d4c6e3a4a)

The current issue is that there is a supporter feature that also wants this real estate in the editor. I think this would be a super helpful addition to have for development but I need to know what to do about the existing code for the chat sidebar that also appears to be WIP?

**Comment by connorhedgehog** [comment]
a toggle, maybe? like a fourth button or another dropdown similar to the theme

**Comment by sushiwt** [comment]
Oh neat! didn't think anybody would pick up on this issue but thank you. I'll keep this open just in case though.

**Comment by BanceDev** [comment]
Yeah I opened a PR and Kyle said he would like to send it through just needs to get a couple more items off his todo to make sure this change is safe CORS-wise. I think it was a great idea, hopefully we can see it on the site here soon :)

**Comment by kyledrake** [comment]
This is queued up. I've been working on some improvements to the CDN this week but I'll get to this when I'm wrapped up with that.
